scrapy/utils/asyncio.py

```diff
@@ -0,0 +1,23 @@
+def install_asyncio_reactor():
+    """ Tries to install AsyncioSelectorReactor
+    """
+    try:
+        import asyncio
+        from twisted.internet import asyncioreactor
+    except ImportError:
+        pass
+    else:
+        from twisted.internet.error import ReactorAlreadyInstalledError
+        try:
+            asyncioreactor.install(asyncio.get_event_loop())
+        except ReactorAlreadyInstalledError:
+            pass
```

Comment:
Similar here. Shoudn’t we call `is_asyncio_reactor_installed` here and raise or log something if the installed reactor does not provide asyncio support? (assuming we can check for this, I’m guessing an alternative asyncio reactor that does not inherit from the one we use may be a problem)